e2e/assets/single-cluster/helm-multi-chart.yaml

```diff
@@ -0,0 +1,8 @@
+kind: GitRepo
+apiVersion: fleet.cattle.io/v1alpha1
+metadata:
+  name: helm
```

Comment:
should this have been `helm-multi-chart`? Also as indicated above, maybe this should be `single-cluster-helm-multi-chart` or `sc-helm-multi-chart`